scripts/statesync.sh

```diff
@@ -0,0 +1,37 @@
+
+#!/bin/bash
+# microtick and bitcanna contributed significantly here.
+set -e
+
+export GOPATH=~/go
+export PATH=$PATH:~/go/bin
+go install ./...
+
+
+# MAKE HOME FOLDER AND GET GENESIS
+osmosisd init test 
+wget -O ~/.osmosisd/config/genesis.json https://cloudflare-ipfs.com/ipfs/QmXRvBT3hgoXwwPqbK6a2sXUuArGM8wPyo1ybskyyUwUxs  
```

Comment:
Is this trying to state sync against mainnet?

Reply:
Yep

Reply:
The way that I'm looking at this is we can use this as an indicator of changes that would cause differences in the app hash, and of changes that might break state sync itself.

Reply:
It will currently fail in the master branch but we can tag it with that one setting that causes it to continue on error.